plugins/serializers/nowmetric/nowmetric.go

```diff
@@ -0,0 +1,152 @@
+package nowmetric
+
+import (
+	"bytes"
+	"encoding/json"
+	"time"
+
+	"github.com/influxdata/telegraf"
+)
+
+type serializer struct {
+	TimestampUnits time.Duration
+}
+
+/*
+Example for the JSON generated and pushed to the MID
+{
+	"metric_type":"cpu_usage_system",
+	"resource":"",
+	"node":"ASGARD",
+	"value": 0.89,
+	"timestamp":1487365430,
+	"ci2metric_id":{"node":"ASGARD"},
+	"source":"Telegraf"
+}
+*/
+
+type OIMetric struct {
+	Metric    string                 `json:"metric_type"`
+	Resource  string                 `json:"resource"`
+	Node      string                 `json:"node"`
+	Value     interface{}            `json:"value"`
+	Timestamp int64                  `json:"timestamp"`
+	CiMapping map[string]interface{} `json:"ci2metric_id"`
```

Comment:
Make this a map[string]string if possible, documentation seems to indicate that this can be a string only.

Reply:
fixed